modules/core/src/main/java/org/apache/synapse/mediators/eip/EIPUtils.java

```
@@ -39,7 +55,15 @@
 public class EIPUtils {
 
     private static final Log log = LogFactory.getLog(EIPUtils.class);
-
+    
+    private static SynapseJsonPath rootJsonPath=null;
+    static{
+    	try{
+    		rootJsonPath=new SynapseJsonPath("$.");
+    	}catch(JaxenException e){
+    	}
```

Do we need to ignore this exception ?